### PR TITLE
support scala 2.12 build

### PR DESCRIPTION
**Pull request by pjfanning**
I have a scenario where we have a 0.8 Kafka Broker managed by a different org but I would like to upgrade my application to Scala 2.12.
I know the 0.8 branch is no longer maintained and all I am looking for is a one-off publish of a scala 2.12 compatible jar from the head of the 0.8 branch.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -137,7 +137,7 @@ subprojects {
   }
 }
 
-for ( sv in ['2_9_1', '2_9_2', '2_10_4', '2_11_5'] ) {
```
Comment (pjfanning) [inline build.gradle:140]: scalatest_2.12 is only released for 3.x branch and there is no scala 2.9.x releases for scalatest 3.x

**Comment by guozhangwang** [comment]
test this.

**Comment by pjfanning** [comment]
@guozhangwang One test fails when I run them with scala 2.12: https://travis-ci.org/scala212-forks/kafka

**Comment by guozhangwang** [comment]
@pjfanning Actually I was telling jenkins to re-trigger builds for the unit tests :) But anyways, seems those jenkins builds failed either. May need to investigate more.

**Comment by guozhangwang** [comment]
test this please.

**Comment by ijuma** [comment]
Given how old 0.8.2 is and the fact that we haven't published Scala 2.12 JARs for 0.9.0.x or 0.10.0.x, it seems unlikely that we'll publish Scala JARs for 0.8.2.

**Comment by pjfanning** [comment]
I have published a scala 2.12 jar to maven central using a forked repo:
com.github.scala212-forks:kafka_2.12:0.8.2.2

**Pull request closed without merging**